test/widgets/compose_box_test.dart

```diff
@@ -37,6 +37,13 @@ import 'test_app.dart';
 void main() {
   TestZulipBinding.ensureInitialized();
 
+  test('lerp light to dark, no crash', () {
+    final a = ComposeBoxTheme.light;
+    final b = ComposeBoxTheme.dark;
+
+    check(() => a.lerp(b, 0.5)).returnsNormally();
+  });
+
```

Comment:
Let's wrap this in a `group('ComposeBoxTheme'` and put it near the other `group`s.